OpenEdXMobile/res/values/text_styles.xml

```diff
@@ -162,17 +163,18 @@
     </style>
 
     <!-- Course Card styles -->
-    <style name="course_card_title" parent="@style/regular_primary_base_text">
+    <style name="course_card_title" parent="@style/bold_edx_black_text">
         <item name="android:layout_width">wrap_content</item>
         <item name="android:layout_height">wrap_content</item>
-        <item name="android:layout_marginBottom">3dp</item>
-        <item name="android:textSize">@dimen/edx_x_large</item>
+        <item name="android:textSize">@dimen/edx_small</item>
+        <item name="android:lineHeight" tools:ignore="NewApi">@dimen/course_card_line_height</item>

```

Comment:
`lineHeight is available for pre-28 API levels only, and also it's working as expected. Can you please look into detail a bit more?

Reply:
Yes, you are right it's working as expected even without `lineHeight`. 
So I have removed it.